src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -688,6 +725,7 @@ def __call__(
         callback_steps: int = 1,
         clean_caption: bool = True,
         use_resolution_binning: bool = True,
+        model_token_max_length: int = 120,
```

Comment:
```suggestion
        max_sequence_length: int = 120,
```

Let's use this variable name throughout? 

Can we also add this argument to the call docstrings?